src/mapeo-manager.js

```diff
@@ -538,10 +547,10 @@ export class MapeoManager extends TypedEmitter {
    * @param {{ waitForSync?: boolean }} [opts] For internal use in tests, set opts.waitForSync = false to not wait for sync during addProject()
    * @returns {Promise<string>}
    */
-  async addProject(
+  addProject = async (
```

Comment:
Any reason for binding this method to `this`?

Reply:
Yes, this line above:

https://github.com/digidem/mapeo-core-next/blob/dd18fea272ec2d0d1b09f4e006f8a190f9d067cf/src/mapeo-manager.js#L163

Happy to change it back if you like.